source/extensions/common/wasm/wasm.cc

```diff
@@ -441,6 +441,10 @@ uint32_t httpCallHandler(void* raw_context, uint32_t uri_ptr, uint32_t uri_size,
 
 uint32_t getTotalMemoryHandler(void*) { return 0x7FFFFFFF; }
 uint32_t _emscripten_get_heap_sizeHandler(void*) { return 0x7FFFFFFF; }
+void _llvm_trapHandler(void*) {
+  std::cerr << "llvm_trapHandler";
```

Comment:
cerr? It's the first time I see the usage of sderr in the code but you may be right

Reply:
Debugging code, removing...